src/etc/poudriere.conf.sample

```diff
@@ -309,3 +309,9 @@ DISTFILES_CACHE=/usr/ports/distfiles
 # processing of the queue slightly, especially for bulk -a builds.
 # Default: no
 #HTML_TRACK_REMAINING=yes
+
+# Have pkg create hashed versions of the pkg filenames with symlinks to
+# original pkg names. The packagesite.yaml file will point to the hashed version
+# of these files. This is usefaully in a cdn enviroment to ensure pkg names are
```

Comment:
s/faully/ful/ && s/enviroment/environment/

Maybe rephrase this comment as:

By using hashed pkg filenames, this allows users to lazily synchronise packages without conflicting with the current packages, for example using rsync or CDNs.


I must admit I'm not clear on the exact steps at the end to switch the package repos over, but maybe this doesn't matter in practice?

Reply:
Thanks. Sorry i shouldn't code at 2am but I just wanted to get a review started for feedback if I was even doing this the right way.